sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/PushTokenApiRequest.kt

```diff
@@ -0,0 +1,37 @@
+package com.klaviyo.analytics.networking.requests
```

Comment:
Left this comment on another PR, but since you are actually moving things around here, thoughts on this going into the `push` module?

Reply:
So my take here was we publish a `push-fcm` module with the intent that it isolates _just_ the FCM dependency in an otherwise very small package. `analytics` contains all Klaviyo API client code, even the push token endpoint. The main reason was that implementing another push platform (fire os I think? maybe baidu?) would just be another module like `push-fireos`

Reply:
Aah gotcha that makes a lot of sense! I'm ok with that approach!